BuildLoader.py

```diff
@@ -339,13 +340,24 @@ def update_fit_table (self):
 			if len(u_code_images) > 0:
 				raise Exception('  Insufficient uCode entries in FIT. Need %d more.' % len(u_code_images))
 
-		if self._board.ACM_SIZE > 0:
+			acm3_index = 0
+			if self._board.ACM_SIZE == 0:
+				acm3_index       = num_fit_entries
+				num_fit_entries     += 1
```

Comment:
No need to have a dependency on UCODE.
Instead, update acm3_index and num_fit_entries after line#406 (else:) if ACM3_SIZE > 0.

Reply:
Ok, will update it